pages/common/docker-compose.md

```diff
@@ -0,0 +1,27 @@
+# docker-compose
+
+> Run and manage multi container docker applications.
+
+- Create and start all containers in the background using a local `docker-compose.yml` file:
+
+`docker-compose up -d`
+
+- Start all containers, rebuild if necessary:
+
+`docker-compose up --build`
+
+- Start all containers using a custom configuration file:
```

Comment:
Just a quick question - how is this file different from the `docker-compose.yml` file ?

Reply:
When using the `--file` param, you can specify any compose file, without it `docker-compose` will look for a file named `docker-compose.yml` inside the working dir

Reply:
Ah, so its the same file. Please change it to "Start all containers using a given `docker-compose.yml` file." 

And change the token in the example to `{{path/to/file}}` to keep it consistent with other pages.

Reply:
sure, though I do want to emphasise that this file doesn't have to be called `docker-compose.yml`

Reply:
Ah I see. So if you don't mention anything. It looks for a file named "docker-compose.yml" by default in the current directory. And if you want to specify some other file, it can have any name.

I was going to comment to change the first description. But I like your new change. That works too.